Tests/DangerTests/GitLabMilestoneTests.swift

```diff
@@ -0,0 +1,37 @@
+@testable import Danger
+import DangerFixtures
+import XCTest
+
+class GitLabMilestoneTests: XCTestCase {
```

Comment:
```suggestion
final class GitLabMilestoneTests: XCTestCase {
```

Reply:
Sorry for forgot to make it final every time 😢